tests/epyccel/test_epyccel_functions.py

```diff
@@ -191,6 +191,31 @@ def f10(x):
     f(x_expected)
     assert np.array_equal(x, x_expected)
 
+def test_multiple_returns_f11():
+    @types('int', 'int', results='int')
+    def ackermann(m, n):
+        if m == 0:
+            return n + 1
+        elif n == 0:
+            return ackermann(m - 1, 1)
+        else:
+            return ackermann(m - 1, ackermann(m, n - 1))
+
+    f = epyccel(ackermann)
+    assert abs(f(2,3)-ackermann(2,3))<1e-16
```

Comment:
The two results should be exactly identical, as they are integer numbers...

Reply:
fixed